src/app/src/main/kotlin/org/imperial/mrc/mint/controllers/MINTErrorController.kt

```diff
@@ -0,0 +1,36 @@
+package org.imperial.mrc.mint.controllers
+
+import org.imperial.mrc.mint.AppProperties
+import org.imperial.mrc.mint.httpStatusFromCode
+import org.springframework.boot.web.servlet.error.ErrorController
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Controller
+import org.springframework.ui.set
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.servlet.ModelAndView
+import javax.servlet.RequestDispatcher
+import javax.servlet.http.HttpServletRequest
+
+@Controller
+class MINTErrorController(private val appProperties: AppProperties) : ErrorController {
+
+    override fun getErrorPath(): String {
+        return "/error"
+    }
+
+    @RequestMapping("/error")
+    fun handleError(request: HttpServletRequest): ModelAndView {
+        val status = request.getAttribute(RequestDispatcher.ERROR_STATUS_CODE)
+                ?.toString()?.toInt()?: 0
```

Comment:
Codecov is complaining about not testing the null case here I think

Reply:
Yes, although I am testing the null case:
https://github.com/mrc-ide/mint/pull/57/files#diff-102c4fb85bfb825bcb76912a6590c4148f709fd4992e095b757b5ae606f6a31eR61

 But it's not smart enough to work out that if the attribute is not null, then the subsequent string and int must also be not null.